ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsTreeStreamReader.scala

```diff
@@ -150,11 +150,10 @@ class TransactionsTreeStreamReader(
         maxParallelPayloadQueries: Int,
         metric: DatabaseMetrics,
     ): Source[EventStorageBackend.Entry[Raw.TreeEvent], NotUsed] = {
+      // Akka requires for this buffer's size to be a power of two.
```

Comment:
Otherwise this will happen:
```
java.lang.IllegalArgumentException: buffer size must be a power of two
	at akka.stream.impl.fusing.ActorGraphInterpreter$BatchingActorInputBoundary.<init>(ActorGraphInterpreter.scala:128)
	at akka.stream.impl.GraphStageIsland.takePublisher(PhasedFusingActorMaterializer.scala:780)
	at akka.stream.impl.IslandTracking.wireOut(PhasedFusingActorMaterializer.scala:345)
	at akka.stream.impl.PhasedFusingActorMaterializer.wireOutlets(PhasedFusingActorMaterializer.scala:600)
	at akka.stream.impl.PhasedFusingActorMaterializer.materialize(PhasedFusingActorMaterializer.scala:512)
	at akka.stream.impl.SubFusingActorMaterializerImpl.materialize(ActorMaterializerImpl.scala:124)
	at akka.stream.impl.fusing.GraphStages$FutureFlattenSource$$anon$7.$anonfun$onFutureSourceCompleted$1(GraphStages.scala:351)
	at akka.stream.impl.fusing.GraphStages$FutureFlattenSource$$anon$7.$anonfun$onFutureSourceCompleted$1$adapted(GraphStages.scala:349)
	at scala.util.Success.map(Try.scala:262)
	at akka.stream.impl.fusing.GraphStages$FutureFlattenSource$$anon$7.onFutureSourceCompleted(GraphStages.scala:349)
	at akka.stream.impl.fusing.GraphStages$FutureFlattenSource$$anon$7.$anonfun$preStart$2(GraphStages.scala:309)
	at akka.stream.impl.fusing.GraphStages$FutureFlattenSource$$anon$7.$anonfun$preStart$2$adapted(GraphStages.scala:309)
	at akka.stream.impl.fusing.GraphInterpreter.runAsyncInput(GraphInterpreter.scala:467)
	at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:517)
	at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:625)
	at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:800)
	at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:818)
	at akka.actor.Actor.aroundReceive(Actor.scala:537)
	at akka.actor.Actor.aroundReceive$(Actor.scala:535)
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundReceive(ActorGraphInterpreter.scala:716)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:580)
	at akka.actor.ActorCell.invoke(ActorCell.scala:548)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)
	at akka.dispatch.Mailbox.run(Mailbox.scala:231)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)

```